extensions/Webhooks/template/en/default/account/prefs/webhooks.html.tmpl

```diff
@@ -0,0 +1,182 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[%# initialize product to component mapping #%]
+
+[% SET selectable_products = user.get_selectable_products %]
+[% SET dont_show_button = 1 %]
+
+<script [% script_nonce FILTER none %]>
+var useclassification = false;
+var first_load = true;
+var last_sel = [];
+var cpts = new Array();
+[% n = 0 %]
+[% FOREACH prod = selectable_products %]
+  cpts['[% n %]'] = [
+    [%- FOREACH comp = prod.components %]'[% comp.name FILTER js %]'[% ", " UNLESS loop.last %] [%- END -%] ];
+  [% n = n + 1 %]
+[% END %]
+</script>
+
+<script src="[% 'js/productform.js' FILTER version FILTER html %]"></script>
+
+<script [% script_nonce FILTER none %]>
+function onSelectProduct() {
+  var component = document.getElementById('component');
+  selectProduct(document.getElementById('product'), component);
+  // selectProduct only supports __Any__ on both elements
+  // we only want it on component, so add it back in
+  try {
+    component.add(new Option('__Any__', ''), component.options[0]);
+  } catch(e) {
+    // support IE
+    component.add(new Option('__Any__', ''), 0);
+  }
+  if ('[% add_component FILTER js %]' != ''
+      && bz_valueSelected(document.getElementById('product'), '[% add_product FILTER js %]')
+  ) {
+    var index = bz_optionIndex(document.getElementById('component'), '[% add_component FILTER js %]');
+    if (index != -1)
+      document.getElementById('component').options[index].selected = true;
+  }
+  onSelectComponent();
+}
+
+function onSelectComponent() {
+  var product_select = document.getElementById('product');
+  var product = product_select.options[product_select.selectedIndex].value;
+  var component = document.getElementById('component').value;
+}
+
+function onRemoveChange() {
+  var cbs = document.getElementById('webhooks_table').getElementsByTagName('input');
+  for (var i = 0, l = cbs.length; i < l; i++) {
+    if (cbs[i].checked) {
+      document.getElementById('remove').disabled = false;
+      return;
+    }
+  }
+  document.getElementById('remove').disabled = true;
+}
+
+window.onload = function() {
+  onSelectComponent();
+  onSelectProduct();
+  onRemoveChange();
+};
+</script>
+
+<p>
+  It will be sent a POST request with the information of the [% terms.bugs %] that match with the events and filters selected to your URL.<br>
+  Documentation about webhooks is available <a href="">here</a>.
```

Comment:
This should be a page.cgi template and point to it.  

Example template
extensions/BMO/template/en/default/pages/etiquette.html.tmpl

The new one would be:
extensions/Webhooks/template/en/default/pages/webhooks.html.tmpl

We will add any user docs there and access the page using `page.cgi?id=webhooks.html`

Reply:
I added the page template and the link but the page content isn't complete. I will work on that after I finish the Push Connector.